robotiq_s_model_articulated_gazebo/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package>
+  <name>robotiq_s_model_articulated_gazebo</name>
+  <version>1.0.0</version>
+  <description>Launch files for spawning the robotiq s-model articulated gripper</description>
+
+  <maintainer email="dash@clearpathrobotics.com">Devon Ash</maintainer>
+  <author email="dash@clearpathrobotics.com">Devon Ash</author>
+
+  <license>BSD</license>
```

Comment:
afaict, the `drcsim/robotiq_hand_description` pkg that you based this on is Apache2.0 licensed. IANAL, but I don't think you just relicense it to BSD. Your companies lawyers should know more about this, but I think you'd have to include the Apache2.0 license text for all files that were originally under that license.